VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -46,7 +47,46 @@ def __init__(self,patching, hutil, logger,para_parser):
             errMsg = 'Failed to serialize customSettings with error: %s, stack trace: %s' % (str(e), traceback.format_exc())
             self.logger.log(errMsg, True, 'Error')
             self.isOnlyOSDiskBackupEnabled = False
-
+        # The command lsscsi is used for mapping the LUN numbers to the disk_names
+        self.command = "sudo lsscsi"
+        self.disksToBeIncluded = []
+        self.root_devices = []
+        self.root_mount_points = ['/' , '/boot/efi']
```

Comment:
this would not only help with modularity - but from what i can see none of these command's existence is being explicitly checked before execution. While a `try/except` block would capture the failure we could employ maintainable logic - like installing the package perhaps - if we can are checking pro-actively. It would also reduce our dependency on the error types provided by python which may or may not allow us to distinguish these in a simple way.
Also please have a look at the `main/patch` module. It has explicit paths to the binary of each command in each distro. 
Distro type can be trivially figured out and then going through patch would give us another layer of protection against preventable exceptions.